networking/eltnam/setup_routers.py

```diff
@@ -0,0 +1,33 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import sys
+import time
+
+teamNum = sys.argv[1]
+
+wd = webdriver.Firefox()
```

Comment:
This script is supposed to be run on a dev computer, not the Beaglebones. Firefox is required, but assuming that most of them are running some Ubuntu derivative, it should be there by default. 

No internet is expected since they need to be connected to the router already.